api/communication-api/src/main/java/com/cognifide/aet/communication/api/metadata/Comparator.java

```diff
@@ -49,6 +49,9 @@ public void setStepResult(ComparatorStepResult stepResult) {
   }
 
   public List<Operation> getFilters() {
+    if(filters == null){
+      return new ArrayList<>();
+    }
```

Comment:
It could be a little be confusing why I compare filters with null, filters have initial value new ArrayList<> so they never are null, but if the array is empty, in the database we haven't got information about it, this field doesn't exist. It was the easiest way to solve NullPointerException, what do you think about saving filters:[] in the database when the array is empty? Or maybe you know another way to solve it?